### PR TITLE
#464 create pull request template

### PR DESCRIPTION
**Pull request by Kobraz**
## Describe your changes:
I created a new PULL_REQUEST_TEMPLATE.md under .github on root.

I added Title, Descriptiopn, Type of changes, Checklist, Issues, Screenshots, Additional information, Feedback

## Type of changes
I created the file from scratch.

## Checklist before requesting a review

* [x] Did you create a new branch before starting with this ticket?
* [x] Did you comment the issue after finish in the Frontend Team Board?
* [x] Did you move the card to Quality Assurance after finish?

## Did you run into any issues?

* [x] No
* [ ] Yes I will describe them below:

## Screenshots:
| Before | After |
---------|--------
| There was nothing before  |  image     |

## Additional information:
I have chatted with Kyrre who made this ticket and he 

## Feedback

* [x] I want feedback on this

### CODE REVIEW

**Comment by kyrregjerstad** [comment]
Great work! 👏
I suggest making the screenshots section optional. It's mainly valuable when the PR includes UI changes. Including code screenshots isn't very helpful IMO.

I'm looking forward to hearing others' thoughts on this, as we'll all be using this template! 🚀

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+## Checklist before requesting a review
+
+* [ ] I created a new branch before starting with this ticket?
+* [ ] I commented the issue after finish in the Frontend Team Board?
+* [ ] I moved the card to Quality Assurance after finish?
+
+## Did you run into any issues?
+
+* [ ] No
+* [ ] Yes I will describe them below:
```
Comment (Olekrr) [inline .github/PULL_REQUEST_TEMPLATE.md:19]: When using checkboxes in markdown, GitHub interprets each checkbox as a task and uses it to track progress as can be seen here in your PR - it gets marked as 5 of 6 tasks completed.

image

I would suggest not using checkboxes for this section, and instead using descriptive text
Something like this perhaps:

image

**Review by Sovak3441 — COMMENTED** [review]
I would love to see this being used across the board!!
I agree tho to the screenshots being optional.

**Review by Olekrr — CHANGES_REQUESTED** [review]
Overall this looks fantastic!
I have suggested one change for the "Did you run into any issues?" section.

I also agree that the screenshots should be optional

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (Kobraz) [reply .github/PULL_REQUEST_TEMPLATE.md:19]: Oh, I wasn't aware that all boxes need to be checked off to achieve 100% completion. I will fix this.

**Comment by Kobraz** [comment]
Have done some changes based on your feedback:

image

**Review by Olekrr — APPROVED** [review]

**Comment by kyrregjerstad** [comment]
> Have done some changes based on your feedback:
> 
> image

looks great! Little GitHub tip for the future: You don't need to upload a screenshot of your changes, your latest commit is visible directly in the 'Files Changed' tab. Here it's also possible to comment directly on the code, and you'll receive the comments straight into vscode as well :)

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+## Title
```
Comment (kyrregjerstad) [inline .github/PULL_REQUEST_TEMPLATE.md:1]: Good job!

Comment (kyrregjerstad) [reply .github/PULL_REQUEST_TEMPLATE.md:1]: like this, here you can see the comment directly in vscode

Comment (Kobraz) [reply .github/PULL_REQUEST_TEMPLATE.md:1]: Ok :) - I wish there was any soulution for radibuttons in Markdown, but for now you have to remove the answer you dont need. Or we could just have inserted ...

Comment (kyrregjerstad) [reply .github/PULL_REQUEST_TEMPLATE.md:1]: True, radio buttons would be awesome in markdown! But I think the solution you came up with is a great way around it

**Pull request merged**